knowledge/responses/i_love_how_animals_taste.md

```diff
@@ -1,7 +1,11 @@
-When it boils down to it, the number one reason people eat animal products is because they enjoy the taste - I know I certainly did, in fact like most people I loved the taste of animal products. I think one of the most disconcerting things about the taste excuse though, is that it is an excuse that bluntly admits that the personal desires of an individual’s taste preference matter more than the morality surrounding an animal’s life and unquestionably horrific death. 
+When it boils down to it, the reason people eat animal products is because they enjoy the taste - in fact, most vegans loved the taste of animal products before going vegan. But sensory pleasure is a poor excuse. It bluntly admits that an individual’s taste preference matter more than the morality surrounding an animal’s life and unquestionably horrific death. 
 
-The question is: “do you value your taste buds higher than the life of an animal”? If the answer to this is yes, one of the big issues with this excuse is that it seeks to validate a non-vegan diet by claiming that we shouldn’t be held responsible for our immoral activities because our selfish impulses are too strong to be suppressed and as such, we can’t be held morally accountable for the actions that we make. But where do we draw the line? Would this argument hold up in court if a murderer in their own defence said, “but I could never give up murdering because I enjoy it too much.”? It requires more than sensory pleasure to morally justify something - and the inconvenience people go through by giving up cheese, for example, is not even remotely comparable to the pain a dairy cow goes through being forcibly impregnated repeatedly, having her children taken away from her and being painfully pumped for her milk before she becomes too weak and is sent off to be slaughtered. 
+The question is: “do you value your taste buds higher than the life of an animal”? And it is not only about the taking of the animal's life. To reach your plate and give you a few minutes of pleasure, animals are forcefully in-bred, live horrendous lives of constant abuse, inhabit bodies that no longer work properly due to genetical modification, and are finally taken to death camps where they see their peers being killed one by one. This is all in accordance with "humane" regulations.
```

Comment:
Though this may seem obvious and well-known to us, I would suggest adding a source that describes the horrendous conditions faced by farmed animals.